# Ukrainian translation for gdebi
# Copyright (c) 2009 Rosetta Contributors and Canonical Ltd 2009
# This file is distributed under the same license as the gdebi package.
# FIRST AUTHOR <EMAIL@ADDRESS>, 2009.
#
msgid ""
msgstr ""
"Project-Id-Version: gdebi\n"
"Report-Msgid-Bugs-To: sebastian.heinlein@web.de\n"
"POT-Creation-Date: 2013-02-09 14:24+0100\n"
"PO-Revision-Date: 2012-07-06 15:01+0000\n"
"Last-Translator: Constantine E. Kozlov <tempor.demonius@gmail.com>\n"
"Language-Team: Ukrainian <uk@li.org>\n"
"Language: uk\n"
"MIME-Version: 1.0\n"
"Content-Type: text/plain; charset=UTF-8\n"
"Content-Transfer-Encoding: 8bit\n"
"X-Launchpad-Export-Date: 2012-10-11 19:24+0000\n"
"X-Generator: Launchpad (build 16130)\n"

#: ../gdebi:48
#, c-format
msgid ""
"usage: %prog [options] filename\n"
"For a graphical version run gdebi-gtk\n"
msgstr ""
"використання: %prog [опції] назва_файлу\n"
"Для графічної версії запустіть gdebi-gtk\n"

#: ../gdebi:54 ../gdebi-gtk:51
msgid "Run non-interactive (dangerous!)"
msgstr "Запустити неінтерактивно (небезпечно!)"

#: ../gdebi:58
msgid "Set an APT configuration option"
msgstr "Встановлення опції налаштування APT"

#: ../gdebi:62
msgid "Do not show progress information"
msgstr "Не показувати прогрес"

#: ../gdebi:66
msgid "Simulate only and print a apt-get install compatible line to stderr"
msgstr "Лише симулювати та вивести рядок, сумісний з apt-get в stderr"

#: ../gdebi:68
msgid "Use alternative root dir"
msgstr "Використовувати іншу теку root"

#: ../gdebi:76
#, c-format
msgid "gdebi error, file not found: %s\n"
msgstr "помилка gdebi, файл не знайдено: %s\n"

#: ../gdebi:95 ../gdebi:104
msgid "Need to be root to install packages"
msgstr "Потрібно бути root, щоб встановлювати пакунки"

#: ../gdebi:106 ../GDebi/GDebiCli.py:153
msgid "Do you want to install the software package? [y/N]:"
msgstr "Ви хочете встановити програмний пакунок? [y/N]:"

#: ../gdebi-gtk:54
msgid "Auto close when the install is finished"
msgstr "Автоматично закривати після завершення інсталяції"

#: ../gdebi-gtk:56
msgid "Use alternative datadir"
msgstr "Використовувати іншу теку даних"

#: ../gdebi-gtk:59
msgid "Remove package"
msgstr "Вилучення пакунка"

#: ../gdebi-gtk:78
msgid "Software index is broken"
msgstr "Індекс програм пошкоджений"

#: ../gdebi-gtk:79
msgid ""
"This is a major failure of your software management system. Please check for "
"broken packages with synaptic, check the file permissions and correctness of "
"the file '/etc/apt/sources.list' and reload the software information with: "
"'sudo apt-get update' and 'sudo apt-get install -f'."
msgstr ""
"Це серйозна помилка у вашій системі керування програмним забезпеченням. "
"Перевірте наявність пошкоджених пакунків за допомогою Synaptic, права "
"доступу на файли та коректність вмісту файлу '/etc/apt/sources.list' та "
"оновіть інформацію про програмне забезпечення за допомогою: 'sudo apt-get "
"update' та 'sudo apt-get install -f'."

#: ../data/gdebi.ui.h:1 ../data/gdebi.desktop.in.h:3
msgid "Install and view software packages"
msgstr "Встановити та оглянути програмні пакунки"

#: ../data/gdebi.ui.h:2
msgid "GPL, see /usr/share/common-licenses/GPL"
msgstr "GPL, див. /usr/share/common-licenses/GPL"

#: ../data/gdebi.ui.h:3
msgid "Terminal"
msgstr "Термінал"

#: ../data/gdebi.ui.h:4
msgid "Automatically close after the changes have been successfully applied"
msgstr "Автоматично закрити після успішного прийняття усіх змін"

#: ../data/gdebi.ui.h:5 ../GDebi/GDebiKDE.py:157 ../GDebi/GDebiKDE.py:159
#: ../GDebi/GDebiKDE.py:321
msgid "Details"
msgstr "Подробиці"

#: ../data/gdebi.ui.h:6 ../GDebi/GDebiKDE.py:320
msgid "<b>To install the following changes are required:</b>"
msgstr "<b>Для встановлення потрібні наступні зміни:</b>"

#: ../data/gdebi.ui.h:7 ../data/gdebi.desktop.in.h:2
msgid "Package Installer"
msgstr "Встановлювач пакунків"

#: ../data/gdebi.ui.h:8
msgid "_File"
msgstr "_Файл"

#: ../data/gdebi.ui.h:9
msgid "_Open…"
msgstr "_Відкрити…"

#: ../data/gdebi.ui.h:10
msgid "_Refresh"
msgstr "_Оновити"

#: ../data/gdebi.ui.h:11
msgid "_Edit"
msgstr "_Редагувати"

#: ../data/gdebi.ui.h:12
msgid "_Help"
msgstr "_Довідка"

#. first, we load all the default descriptions -- pyuic doesn't use
#. gettext as default (FIXME, copy code from language-selector)
#: ../data/gdebi.ui.h:13 ../GDebi/GDebiKDE.py:155
msgid "Package:"
msgstr "Пакунок:"

#: ../data/gdebi.ui.h:14
msgid "<b><big>   </big></b>"
msgstr "<b><big>   </big></b>"

#: ../data/gdebi.ui.h:15
msgid "_Details"
msgstr "_Подробиці"

#: ../data/gdebi.ui.h:16 ../GDebi/GDebiKDE.py:156
msgid "Status:"
msgstr "Стан:"

#: ../data/gdebi.ui.h:17
msgid "Description:"
msgstr "Опис:"

#. img = Gtk.Image()
#. img.set_from_stock(Gtk.STOCK_APPLY,Gtk.IconSize.BUTTON)
#. self.button_install.set_image(img)
#: ../data/gdebi.ui.h:18 ../GDebi/GDebiGtk.py:335 ../GDebi/GDebiGtk.py:392
msgid "_Install Package"
msgstr "_Встановити пакунок"

#: ../data/gdebi.ui.h:19
msgid "_Remove Package"
msgstr "_Видалити пакет"

#: ../data/gdebi.ui.h:20
msgid "_Download Package"
msgstr "_Завантажити пакет"

#: ../data/gdebi.ui.h:21 ../GDebi/GDebiKDE.py:158
msgid "Description"
msgstr "Опис"

#: ../data/gdebi.ui.h:22 ../GDebi/GDebiKDE.py:164
msgid "<b>Version:</b>"
msgstr "<b>Версія:</b>"

#: ../data/gdebi.ui.h:23 ../GDebi/GDebiKDE.py:165
msgid "<b>Maintainer:</b>"
msgstr "<b>Упорядник:</b>"

#: ../data/gdebi.ui.h:24 ../GDebi/GDebiKDE.py:166
msgid "<b>Priority:</b>"
msgstr "<b>Пріоритет:</b>"

#: ../data/gdebi.ui.h:25 ../GDebi/GDebiKDE.py:167
msgid "<b>Section:</b>"
msgstr "<b>Секція:</b>"

#: ../data/gdebi.ui.h:26 ../GDebi/GDebiKDE.py:168
msgid "<b>Size:</b>"
msgstr "<b>Розмір:</b>"

#: ../data/gdebi.ui.h:27
msgid "  "
msgstr "  "

#: ../data/gdebi.ui.h:28
msgid "Included files"
msgstr "Включені файли"

#: ../data/gdebi.ui.h:29
msgid "Lintian output"
msgstr ""

#: ../data/gdebi.desktop.in.h:1
msgid "GDebi Package Installer"
msgstr "Встановлювач пакунків GDebi"

#. Translators: it's for missing entries in the deb package,
#. e.g. a missing "Maintainer" field
#: ../GDebi/DebPackage.py:38
#, python-format
msgid "%s is not available"
msgstr "%s є недоступним"

#: ../GDebi/GDebiGtk.py:93
msgid "Copy selected text"
msgstr "Копіювати виділений текст"

#: ../GDebi/GDebiGtk.py:104
msgid "Loading..."
msgstr "Завантаження..."

#: ../GDebi/GDebiGtk.py:179
msgid "Can not download as root"
msgstr "Не можливо завантажити від root"

#: ../GDebi/GDebiGtk.py:180
msgid ""
"Remote packages can not be downloaded when running as root. Please try again "
"as a normal user."
msgstr ""
"Віддалені пакунки не можуть бути завантажені під root’ом. Будь ласка, "
"спробуйте знову під звичайним користувачем"

#: ../GDebi/GDebiGtk.py:193
msgid "Downloading package"
msgstr "Завантаження пакунків"

#: ../GDebi/GDebiGtk.py:200
msgid "Download failed"
msgstr "Завантаження не вдалось"

#: ../GDebi/GDebiGtk.py:201
#, python-format
msgid "Downloading the package failed: file '%s' '%s'"
msgstr "Завантаження не вдалось: файл '%s' '%s'"

#. set window title
#. set name
#: ../GDebi/GDebiGtk.py:249 ../GDebi/GDebiKDE.py:208
#, python-format
msgid "Package Installer - %s"
msgstr "Встановлювач пакунків - %s"

#: ../GDebi/GDebiGtk.py:306
msgid "Package control data"
msgstr "Дані контролю пакунку"

#: ../GDebi/GDebiGtk.py:309
msgid "Upstream data"
msgstr "Вихідні дані"

#: ../GDebi/GDebiGtk.py:315
msgid "Error reading filelist"
msgstr "Помилка читання файлу-списку"

#: ../GDebi/GDebiGtk.py:331
msgid "Error: "
msgstr "Помилка: "

#: ../GDebi/GDebiGtk.py:346
msgid "Error: no longer provides "
msgstr ""

#: ../GDebi/GDebiGtk.py:362
msgid "Same version is already installed"
msgstr "Така ж версія вже встановлена"

#: ../GDebi/GDebiGtk.py:363
msgid "_Reinstall Package"
msgstr "_Перевстановити пакунок"

#: ../GDebi/GDebiGtk.py:401
msgid ""
"No lintian available.\n"
"Please install using sudo apt-get install lintian"
msgstr ""

#: ../GDebi/GDebiGtk.py:404
msgid "Running lintian..."
msgstr ""

#: ../GDebi/GDebiGtk.py:421
#, python-format
msgid ""
"\n"
"Lintian finished with exit status %s"
msgstr ""

#: ../GDebi/GDebiGtk.py:449
msgid "Selection is a directory"
msgstr "Обрано каталог"

#: ../GDebi/GDebiGtk.py:454 ../GDebi/GDebiGtk.py:460
#, python-format
msgid "Error reading file content '%s'"
msgstr "Помилка читання вмісту файлу '%s'"

#: ../GDebi/GDebiGtk.py:465
msgid "File content can not be extracted"
msgstr "Не можливо видобути вміст файлу"

#: ../GDebi/GDebiGtk.py:476
#, python-format
msgid "<b>To be removed: %s</b>"
msgstr "<b>Буде видалено: %s</b>"

#: ../GDebi/GDebiGtk.py:478 ../GDebi/GDebiKDE.py:314
#, python-format
msgid "To be installed: %s"
msgstr "Буде встановлено: %s"

#: ../GDebi/GDebiGtk.py:493
msgid "Open Software Package"
msgstr "Відкрити програмний пакунок"

#: ../GDebi/GDebiGtk.py:498
msgid "Software packages"
msgstr "Програмні пакунки"

#: ../GDebi/GDebiGtk.py:547
msgid "Dependency problems"
msgstr "Проблеми залежностей"

#: ../GDebi/GDebiGtk.py:548
#, python-format
msgid "One or more packages are required by %s, it cannot be removed."
msgstr ""
"Пакет \"%s\" вимагається одним або більше пакетами і не може бути видалений."

#: ../GDebi/GDebiGtk.py:556
msgid "File not found"
msgstr "Файл не знайдено"

#: ../GDebi/GDebiGtk.py:557
msgid "You tried to install a file that does not (or no longer) exist. "
msgstr "Ви спробували встановити файл, якого не існує (або більше не існує). "

#: ../GDebi/GDebiGtk.py:568
msgid "Installing package file..."
msgstr "Встановлення файла пакунка..."

#: ../GDebi/GDebiGtk.py:570
msgid "Removing package..."
msgstr "Видалення пакету..."

#: ../GDebi/GDebiGtk.py:573
msgid "Install unauthenticated software?"
msgstr "Встановити недостовірне програмне забезпечення?"

#: ../GDebi/GDebiGtk.py:574
msgid ""
"Malicious software can damage your data and take control of your system.\n"
"\n"
"The packages below are not authenticated and could therefore be of malicious "
"nature."
msgstr ""
"Шкідливе програмне забезпечення може пошкодити дані і взяти контроль над "
"системою.\n"
"\n"
"Пакунки, перелічені нижче, не ідентифіковані, а отже можуть містити шкідливе "
"ПЗ."

#: ../GDebi/GDebiGtk.py:602
msgid "You need to grant administrative rights to install software"
msgstr ""
"Щоб встановити програмне забезпечення, ви мусите володіти правами "
"адміністратора"

#: ../GDebi/GDebiGtk.py:603
msgid ""
"\n"
"It is a possible security risk to install packages files manually.\n"
"Install software from trustworthy software distributors only.\n"
msgstr ""
"\n"
"Встановлення пакунків вручну, ймовірно, складає ризик безпеці.\n"
"Встановлюйте програмне забезпечення лише від тих постачальників, яким "
"довіряєте.\n"

#: ../GDebi/GDebiGtk.py:608
msgid "You need to grant administrative rights to remove software"
msgstr "Необхідно надати права адміністрування для видалення пакетів"

#: ../GDebi/GDebiGtk.py:609
msgid "It is a possible risk to remove packages."
msgstr ""

#: ../GDebi/GDebiGtk.py:631 ../GDebi/GDebiGtk.py:695
msgid "Failed to install package file"
msgstr "Не вдалося встановити файл пакунка"

#: ../GDebi/GDebiGtk.py:633
msgid "Failed to remove package"
msgstr "Не вдалось вилучити пакет"

#. errMsg = "%s" % msg
#: ../GDebi/GDebiGtk.py:679 ../GDebi/GDebiKDE.py:370
msgid "Could not download all required files"
msgstr "Не вдалося завантажити всі необхідні файли"

#: ../GDebi/GDebiGtk.py:680 ../GDebi/GDebiKDE.py:371
msgid "Please check your internet connection or installation medium."
msgstr "Будь ласка, перевірте ваше інтернет-з'єднання або носій встановлення."

#: ../GDebi/GDebiGtk.py:684 ../GDebi/GDebiKDE.py:376
msgid "Could not install all dependencies"
msgstr "Не вдалося встановити всі залежності"

#: ../GDebi/GDebiGtk.py:685 ../GDebi/GDebiKDE.py:377
msgid ""
"Usually this is related to an error of the software distributor. See the "
"terminal window for more details."
msgstr ""
"Зазвичай це вважають помилкою постачальника програмного забезпечення. За "
"подробицями див. у вікно терміналу"

#: ../GDebi/GDebiGtk.py:701
#, python-format
msgid "Installing %s"
msgstr "Встановлюю %s"

#: ../GDebi/GDebiGtk.py:704
#, python-format
msgid "Removing %s"
msgstr "Видаляється %s"

#. self.label_action.set_markup("<b><big>"+_("Package installed")+"</big></b>")
#. show the button
#. self.button_deb_install_close.set_sensitive(True)
#. self.button_deb_install_close.grab_default()
#: ../GDebi/GDebiGtk.py:726 ../GDebi/GDebiKDE.py:396
msgid "Installation finished"
msgstr "Встановлення завершено"

#: ../GDebi/GDebiGtk.py:728
msgid "Removal finished"
msgstr "Видалення завершене"

#: ../GDebi/GDebiGtk.py:731
#, python-format
msgid "Package '%s' was installed"
msgstr "Пакунок \"%s\" було встановлено"

#: ../GDebi/GDebiGtk.py:733
#, python-format
msgid "Package '%s' was removed"
msgstr "Пакет \"%s\" було вилучено"

#: ../GDebi/GDebiGtk.py:736 ../GDebi/GDebiKDE.py:400
#, python-format
msgid "Failed to install package '%s'"
msgstr "Не вдалося встановити пакунок \"%s\""

#: ../GDebi/GDebiGtk.py:739
#, python-format
msgid "Failed to remove package '%s'"
msgstr "Не вдалось вилучити пакет \"%s\""

#: ../GDebi/GDebiGtk.py:743
msgid "Installation complete"
msgstr "Встановлення завершено"

#: ../GDebi/GDebiGtk.py:745
msgid "Removal complete"
msgstr "Видалення завершене"

#: ../GDebi/GDebiGtk.py:753 ../GDebi/GDebiKDE.py:410
msgid "Failed to completely install all dependencies"
msgstr "Не вдалося повністю встановити всі залежності"

#: ../GDebi/GDebiGtk.py:755
msgid "Failed to completely remove package"
msgstr "Не вдалось повністю видалити пакет"

#: ../GDebi/GDebiGtk.py:756 ../GDebi/GDebiKDE.py:411
msgid "To fix this run 'sudo apt-get install -f' in a terminal window."
msgstr ""
"Щоб виправити це, запустіть \"sudo apt-get install -f\" у вікні терміналу."

#. ui
#: ../GDebi/GDebiGtk.py:875 ../GDebi/KDEAptDialogs.py:70
#, python-format
msgid "Installing '%s'..."
msgstr "Встановлення \"%s\"..."

#: ../GDebi/GDebiGtk.py:878
#, python-format
msgid "Removing '%s'..."
msgstr "Видалення пакету «%s»..."

#: ../GDebi/GDebiGtk.py:982 ../GDebi/KDEAptDialogs.py:138
msgid "Installing dependencies..."
msgstr "Встановлення залежностей..."

#: ../GDebi/GDebiGtk.py:1027 ../GDebi/KDEAptDialogs.py:201
#: ../GDebi/KDEAptDialogs.py:214 ../GDebi/KDEAptDialogs.py:216
msgid "Downloading additional package files..."
msgstr "Завантаження додаткових файлів пакунків..."

#: ../GDebi/GDebiGtk.py:1036 ../GDebi/KDEAptDialogs.py:214
#, python-format
msgid "File %s of %s at %sB/s"
msgstr "Файл %s з %s при %sБ/с"

#: ../GDebi/GDebiGtk.py:1038 ../GDebi/KDEAptDialogs.py:216
#, python-format
msgid "File %s of %s"
msgstr "Файл %s з %s"

#. print "mediaChange %s %s" % (medium, drive)
#: ../GDebi/GDebiGtk.py:1045 ../GDebi/KDEAptDialogs.py:221
#, python-format
msgid "Please insert '%s' into the drive '%s'"
msgstr "Будь ласка, вставте \"%s\" у привід \"%s\""

#: ../GDebi/GDebiCli.py:57
msgid "Configuration items must be specified with a =<value>\n"
msgstr "Конфігураційні пункти мають бути задані разом із =<value>\n"

#: ../GDebi/GDebiCli.py:63
#, python-format
msgid "Couldn't set APT option %s to %s\n"
msgstr "Неможливо змінити опцію APT %s на %s\n"

#: ../GDebi/GDebiCli.py:76
#, python-format
msgid "Unknown package type '%s', exiting\n"
msgstr "Невідомий тип пакунка \"%s\", виходжу\n"

#: ../GDebi/GDebiCli.py:80
msgid "Failed to open the software package\n"
msgstr "Не вдалося відкрити пакунок програмного забезпечення\n"

#: ../GDebi/GDebiCli.py:81
msgid ""
"The package might be corrupted or you are not allowed to open the file. "
"Check the permissions of the file.\n"
msgstr ""
"Пакунок міг бути пошкодженим або Вам не дозволено відкрити цей файл. "
"Перевірте дозволи файла.\n"

#: ../GDebi/GDebiCli.py:87
msgid "This package is uninstallable\n"
msgstr "Цей пакунок є невстановлюваним\n"

#: ../GDebi/GDebiCli.py:96 ../GDebi/GDebiKDE.py:244
msgid "No description is available"
msgstr "Немає жодного опису"

#: ../GDebi/GDebiCli.py:106
msgid "The following packages are UNAUTHENTICATED: "
msgstr "Наступні пакунки є НЕДОСТОВІРНИМИ: "

#: ../GDebi/GDebiCli.py:110
msgid "Requires the REMOVAL of the following packages: "
msgstr "Потребує ВИДАЛЕННЯ наступних пакунків: "

#: ../GDebi/GDebiCli.py:115
msgid "Requires the installation of the following packages: "
msgstr "Потребує встановлення наступних пакунків: "

#: ../GDebi/GDebiCli.py:131
#, python-format
msgid "Error during install: '%s'"
msgstr "Помилка під час встановлення: '%s'"

#: ../GDebi/GDebiKDE.py:160
msgid "Included Files"
msgstr "Включені файли"

#: ../GDebi/GDebiKDE.py:162 ../GDebi/GDebiKDE.py:302
msgid "&Install Package"
msgstr "&Встановити пакунок"

#: ../GDebi/GDebiKDE.py:163
msgid "&Download Package"
msgstr "&Завантажити пакет"

#: ../GDebi/GDebiKDE.py:188
msgid "The package file does not exist"
msgstr "Файл пакунка не існує"

#: ../GDebi/GDebiKDE.py:189
msgid ""
"A nonexistent file has been selected for installation. Please select an "
"existing .deb package file."
msgstr ""
"Для встановлення було вибрано неіснуючий файл. Будь ласка, виберіть існуючий "
"файл пакунка .deb."

#. self.textLabel1_3_2.setText(_("Same version is already installed"))
#: ../GDebi/GDebiKDE.py:270
msgid "&Reinstall Package"
msgstr "&Перевстановити пакунок"

#: ../GDebi/GDebiKDE.py:305
msgid "Re&install Package"
msgstr "Пере&встановити пакунок"

#: ../GDebi/GDebiKDE.py:316
#, python-format
msgid "To be removed: %s"
msgstr "Буде видалено: %s"

#: ../GDebi/GDebiKDE.py:398
msgid "<b>"
msgstr "<b>"

#: ../GDebi/GDebiCommon.py:67
msgid "Broken dependencies"
msgstr "Розбиті залежності"

#: ../GDebi/GDebiCommon.py:68
msgid ""
"Your system has broken dependencies. This application can not continue until "
"this is fixed. To fix it run 'gksudo synaptic' or 'sudo apt-get install -f' "
"in a terminal window."
msgstr ""
"Ваша система має розбиті залежності. Цей додаток не може продовжувати, доки "
"це не буде виправлено. Для виправлення цього запустіть \"gksudo synaptic\" "
"або \"sudo apt-get install -f\" у вікні терміналу"

#: ../GDebi/GDebiCommon.py:86
#, python-format
msgid "'%s' is not a Debian package"
msgstr "\"%s\" не є пакунком Debian"

#: ../GDebi/GDebiCommon.py:87
#, python-format
msgid ""
"The MIME type of this file is '%s' and can not be installed on this system."
msgstr ""
"Тип MIME цього файла є \"%s\" і не може бути встановлений на цій системі."

#: ../GDebi/GDebiCommon.py:91
#, python-format
msgid "Could not open '%s'"
msgstr "Неможливо відкрити \"%s\""

#: ../GDebi/GDebiCommon.py:92
msgid ""
"The package might be corrupted or you are not allowed to open the file. "
"Check the permissions of the file."
msgstr ""
"Пакунок міг бути пошкодженим, або вам не дозволено відкрити файл. Перевірте "
"дозволи файла."

#: ../GDebi/GDebiCommon.py:111
msgid "Same version is available in a software channel"
msgstr "Така ж версія наявна в каналі програмного забезпечення"

#: ../GDebi/GDebiCommon.py:112
msgid "You are recommended to install the software from the channel instead."
msgstr ""
"Вам рекомендовано натомість встановити це програмне забезпечення з каналу."

#: ../GDebi/GDebiCommon.py:116
msgid "An older version is available in a software channel"
msgstr "Старіша версія доступна в каналі програмного забезпечення"

#: ../GDebi/GDebiCommon.py:117
msgid ""
"Generally you are recommended to install the version from the software "
"channel, since it is usually better supported."
msgstr ""
"Взагалі вам рекомендовано встановити версію з каналу програмного "
"забезпечення, оскільки вона зазвичай краще підтримується."

#: ../GDebi/GDebiCommon.py:122
msgid "A later version is available in a software channel"
msgstr "Пізніша версія доступна в каналі програмного забезпечення"

#: ../GDebi/GDebiCommon.py:124
msgid ""
"You are strongly advised to install the version from the software channel, "
"since it is usually better supported."
msgstr ""
"Ми наполегливо радимо вам встановити версію з каналу програмного "
"забезпечення, оскільки вона краще підтримується."

#: ../GDebi/GDebiCommon.py:166
msgid "All dependencies are satisfied"
msgstr "Всі залежності задоволено"

#. FIXME: use ngettext here
#: ../GDebi/GDebiCommon.py:169
#, python-format
msgid "Requires the <b>removal</b> of %s packages\n"
msgstr "Потребує <b>видалення</b> %s пакунків\n"

#: ../GDebi/GDebiCommon.py:171
#, python-format
msgid "Requires the installation of %s packages"
msgstr "Потребує встановлення %s пакунків"

#: ../GDebi/GDebiCommon.py:179 ../GDebi/GDebiCommon.py:197
msgid "Only one software management tool is allowed to run at the same time"
msgstr ""
"Лише один інструмент керування програмним забезпеченням дозволено запускати "
"одночасно"

#: ../GDebi/GDebiCommon.py:181 ../GDebi/GDebiCommon.py:199
msgid ""
"Please close the other application e.g. 'Update Manager', 'aptitude' or "
"'Synaptic' first."
msgstr ""
"Будь ласка, спершу закрийте інший додаток, як-от 'Менеджер оновлення', "
"'aptitude' чи 'Synaptic'."

#. change = QMessageBox.question(None, _("Media Change"), msg, QMessageBox.Ok, QMessageBox.Cancel)
#: ../GDebi/KDEAptDialogs.py:223
msgid "Media Change"
msgstr "Зміна Носія"

#~ msgid "Software package"
#~ msgstr "Програмний пакунок"

#~ msgid "Open..."
#~ msgstr "Відкрити..."

#~ msgid "gdebi"
#~ msgstr "gdebi"

#~ msgid ""
#~ "Malicous software can damage your data and take control of your system.\n"
#~ "\n"
#~ "The packages below are not authenticated and could therefor be of "
#~ "malicous nature."
#~ msgstr ""
#~ "Шкідливе програмне забезпечення може пошкодити ваші дані та захопити "
#~ "контроль над вашою системою.\n"
#~ "\n"
#~ "Пакунки, наведені нижче, не є достовірними, а тому можуть мати шкідливу "
#~ "природу."

#~ msgid "Dependency is not satisfiable: %s\n"
#~ msgstr "Залежність неможливо задовольнити: %s\n"

#~ msgid "Conflicts with the installed package '%s'\n"
#~ msgstr "Конфліктує з встановленим пакунком \"%s\"\n"

#~ msgid "No Architecture field in the package"
#~ msgstr "В пакунку відсутнє поле \"Архітектура\""

#~ msgid "Wrong architecture '%s'"
#~ msgstr "Неправильна архітектура \"%s\""

#~ msgid "A later version is already installed"
#~ msgstr "Пізніша версія вже встановлена"

#~ msgid "Breaks exisiting package '%s' conflict: %s (%s %s)\n"
#~ msgstr "Розбиває існуючий конфлікт пакунка \"%s\": %s (%s %s)\n"

#~ msgid "Breaks exisiting package '%s' dependency %s (%s %s)\n"
#~ msgstr "Розбиває існуючу залежність пакунка \"%s\" %s (%s %s)\n"

#~ msgid "Analysing dependencies"
#~ msgstr "Йде аналіз залежностей"

#~ msgid "Cannot install '%s'"
#~ msgstr "Неможливо встановити \"%s\""

#~ msgid "Failed to satisfy all dependencies (broken cache)"
#~ msgstr "Не вдалося задовольнити всі залежності (розбитий кеш)"

#~ msgid "List of files could not be read, please report this as a bug"
#~ msgstr ""
#~ "Список файлів неможливо прочитати, будь ласка, повідомте про це як про баґ"

#~ msgid "Y"
#~ msgstr "Y"

#~ msgid "translator-credits"
#~ msgstr ""
#~ "Launchpad Contributions:\n"
#~ "  Constantine E. Kozlov https://launchpad.net/~tempor-demonius\n"
#~ "  Luca Falavigna https://launchpad.net/~dktrkranz\n"
#~ "  Maks Lyashuk aka Ma)(imuM https://launchpad.net/~probel\n"
#~ "  Oleg Fedorchuk https://launchpad.net/~enter\n"
#~ "  atany https://launchpad.net/~ye-gorshkov\n"
#~ "  kolybasov https://launchpad.net/~kolybasov\n"
#~ "  mikhalek https://launchpad.net/~mikhalek\n"
#~ "  svv https://launchpad.net/~skrypnychuk\n"
#~ "  Сергій Матрунчик https://launchpad.net/~sergiy.matrunchyk"

#~ msgid "IOError during filelist read: %s"
#~ msgstr "Помилка введення-виведення під час читання списку файлів: %s"

#~ msgid "_Open..."
#~ msgstr "_Відкрити…"
